blog/arquillian-core-1.2.0.Final.textile

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+date: 2017-12-20
+author: dipak-pawar
+title: Welcome JUnit Rules for Arquillian
```

Comment:
Have a look at other release posts - I think only `date` and `author` are necessary.

Title can be wrapped in `h3.` instead